### PR TITLE
fix(summary): add grpc opt for summary

### PR DESCRIPTION
**Pull request by rajaSahil**
Signed-off-by: rajaSahil <sahilraja242@gmail.com>

- Add grpc opt for summary

### CODE REVIEW

**Review by nyrahul — COMMENTED** [review]
I dont see `&summaryOptions.GRPC` getting used anywhere.

**Comment by Ankurk99** [comment]
> I dont see &summaryOptions.GRPC getting used anywhere.
Rahul the GRPC option was already there in the client side: https://github.com/kubearmor/kubearmor-client/blob/main/summary/summary.go#L41-L55, there was just no flag for it to be used

**Comment by seswarrajan** [comment]
> I dont see `&summaryOptions.GRPC` getting used anywhere.

@nyrahul  The option is getting used in summary/summary.go

File: cmd/summary.go [hunk 1]
```diff
@@ -25,6 +25,8 @@ var summaryCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(summaryCmd)
+
+	summaryCmd.Flags().StringVar(&summaryOptions.GRPC, "gRPC", "", "gRPC server information")
```
Comment (Ankurk99) [inline cmd/summary.go:29]: Would "Specify gRPC port" be more verbose?

**Review by seswarrajan — APPROVED** [review]

File: cmd/summary.go (hunk 1 above)
Comment (seswarrajan) [reply cmd/summary.go:29]: @Ankurk99 I don't think is port alone. This will be **`IP:port`** combination. Hence server information LGTM.

**Review by nyrahul — APPROVED** [review]

**Review by Ankurk99 — APPROVED** [review]

**Pull request merged**